commit 6fd8bb8c630d3e43e35fd34f6277a1a4dd0daca9
Author: Norman Maurer <norman_maurer@apple.com>
Date:   Tue Aug 30 06:40:42 2016 +0200

    [#5763] DefaultEventLoopGroup doesn't expose ctor variant that accepts custom Executor
    
    Motivation:
    
    The DefaultEventLoopGroup class extends MultithreadEventExecutorGroup but doesn't expose the ctor variants that accept a custom Executor like NioEventLoopGroup and EpollEventLoopGroup do.
    
    Modifications:
    
    Add missing constructor.
    
    Result:
    
    Be able to use custom Executor with DefaultEventLoopGroup.

diff --git a/transport/src/main/java/io/netty/channel/DefaultEventLoopGroup.java b/transport/src/main/java/io/netty/channel/DefaultEventLoopGroup.java
index 6e8ba13..ee5eeea 100644
--- a/transport/src/main/java/io/netty/channel/DefaultEventLoopGroup.java
+++ b/transport/src/main/java/io/netty/channel/DefaultEventLoopGroup.java
@@ -36,7 +36,7 @@ public class DefaultEventLoopGroup extends MultithreadEventLoopGroup {
      * @param nThreads          the number of threads to use
      */
     public DefaultEventLoopGroup(int nThreads) {
-        this(nThreads, null);
+        this(nThreads, (ThreadFactory) null);
     }
 
     /**
@@ -49,6 +49,16 @@ public class DefaultEventLoopGroup extends MultithreadEventLoopGroup {
         super(nThreads, threadFactory);
     }
 
+    /**
+     * Create a new instance
+     *
+     * @param nThreads          the number of threads to use
+     * @param executor          the Executor to use, or {@code null} if the default should be used.
+     */
+    public DefaultEventLoopGroup(int nThreads, Executor executor) {
+        super(nThreads, executor);
+    }
+
     @Override
     protected EventLoop newChild(Executor executor, Object... args) throws Exception {
         return new DefaultEventLoop(this, executor);
